File: round/round.js
// ATOMIC FUNCTION:round
// LANG: node.js
// FUNCTION: Math.round(val, precision)

var start_val=1.2;
var expected_end_val=1;
var precision=0;
var end_val=Math.round(start_val,precision);

// TEST
if(end_val==expected_end_val){
    console.log('PASS');
}else{
    console.log('FAIL');
}

File: abs/round.py
#!/usr/bin/env python
# coding: utf-8

# In[2]:


# ATOMIC FUNCTION: round
# TAGS: math，numeric
# LANG:python
# FUNCTION round(val, precision)

start_val=1.2
expected_end_val=1
precision=0
end_val=round(start_val,precision)

#Test
if(end_val==expected_end_val):
    print("PASS")

else:
    print("FAIL")


# In[ ]:




